doc/exploring-cheriot-rtos.md

```diff
@@ -0,0 +1,49 @@
+# Exploring CHERIoT RTOS
+## CHERIoT RTOS Orientation
+All the software in this repository runs on the CHERIoT RTOS, which is pulled into this repository as a submodule at [`cheriot-rtos`].
+The [CHERIoT Programmer's Guide] contains most of what a programmer would need to know to use the RTOS.
+
+[`cheriot-rtos`]: ../cheriot-rtos
+[CHERIoT Programmer's Guide]: https://cheriot.org/book/
+
+The different boards supported can be found in [`cheriot-rtos/sdk/boards/`](../cheriot-rtos/sdk/boards), of particular interest will be the Sonata's description in `sonata.json`.
+More on board descriptions can be found in [`cheriot-rtos/docs/BoardDescriptions.md`](../cheriot-rtos/docs/BoardDescriptions.md).
+The drivers (structures that map onto a peripherals' MMIO) and add functionality can be found in [`cheriot-rtos/sdk/include/platform/`](../cheriot-rtos/sdk/include/platform/); the Sonata/Sunburst specific peripherals can be found in `sunburst/` within the aforementioned directory.
+
+To explore the various utility libraries available, look through [`cheriot-rtos/sdk/include/`](../cheriot-rtos/sdk/include/).
+When starting first starting to explore capabilities, the [`CHERI::Capability`](../cheriot-rtos/sdk/include/cheri.hh) class is useful for pointer introspection.
```

Comment:
> When starting first starting to explore capabilities

Typo: When ~starting~ first starting to explore capabilities

Reply:
Addressed in: https://github.com/lowRISC/sonata-software/pull/42